cinema_service/settings.py

```diff
@@ -39,13 +39,15 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
     "rest_framework",
     "cinema",
     "user",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
Consider wrapping the inclusion of `DebugToolbarMiddleware` in `MIDDLEWARE` with a condition to check if `DEBUG` is `True`. This will prevent it from being included in a production environment.